src/desktop/apps/artwork/components/partner_stub/index.styl

```diff
@@ -4,7 +4,6 @@
 
 .artwork-partner-stub
   serif('s-body')
-  border-bottom 1px solid gray-lighter-color
```

Comment:
From my understanding there should only ever be a line between sidebar components, never underneath. Would it be possible to use something like CSS’ adjacent sibling selector instead of adding borders to individual components?

Reply:
(Same applies to the changes above)

Reply:
I actually did do most of the borders with adjacent sibling selector but in this case those styles moved up to commercial component style which render all this smaller templates/components. This whole breakdown is a bit arbitrary now but some of the smaller templates are also used in auction artwork so I left them along.

The several borders that are still there is on parts that are always followed by something but there are too many options what they can be followed with. So I kept the border on the component itself.

Reply:
Okeydokey 👍